scripts/obtain/dotnet-install.ps1

```diff
@@ -281,7 +292,13 @@ function Get-List-Of-Directories-And-Versions-To-Unpack-From-Dotnet-Package([Sys
 function Extract-Dotnet-Package([string]$ZipPath, [string]$OutPath) {
     Say-Invocation $MyInvocation
 
-    Add-Type -Assembly System.IO.Compression.FileSystem | Out-Null
+    try {
```

Comment:
This pattern needs to also be applies to the System.NET.Http logic that was added.  When I tested this, my PS environment already had the assembly loaded therefore it didn't fail on NanoServer.  If you run within a new PS session, it will fail.